packages/monaco-editor-wrapper/src/wrapper.ts

```diff
@@ -29,24 +29,45 @@ export class MonacoEditorLanguageClientWrapper {
     private id: string;
 
     private editorApp: EditorAppClassic | EditorAppExtended | undefined;
-    private languageClientWrapper: LanguageClientWrapper;
+    private languageClientWrapper: LanguageClientWrapper = new LanguageClientWrapper();
     private serviceConfig: InitializeServiceConfig;
     private logger: Logger;
+    private initDone = false;
 
-    private init(userConfig: UserConfig) {
+    async init(userConfig: UserConfig) {
```

Comment:
I would definitely put some function headers on these public-facing methods to detail them a bit more

Reply:
👍 Will do.